### ISSUE TITLE
Album headers 'fade' with the background

### ISSUE THREAD

**Issue by jahirfiquitiva** [dodgingpixels/Optic#26]
What I mean is that, if one chooses the same color for theme and header color, the header 'mixes' with the background, so one cannot really determine which header corresponds to which album

Header colors could be changes to card colors for every theme:
Light -> #ffffff
Dark -> #424242
Amoled -> #212121

**Comment by staber** [comment]
This could also be some peoples desired effect. i.e. choosing light theme & light captions is similar to how local albums appear in Google Photos.

**Comment by jahirfiquitiva** [comment]
@staber 
Yeah, well, that's one of the reasons I don't use Google Photos 🙄😕

**Comment by staber** [comment]
So basically, you're looking for more options or possibly just a color picker.

**Comment by jahirfiquitiva** [comment]
@staber 
That could be good too.
